doc/install.rst

```diff
@@ -89,7 +90,8 @@ PyGMT requires the following libraries:
 The following are optional (but recommended) dependencies:
 
 * `IPython <https://ipython.org/>`__: For embedding the figures in Jupyter
```

Comment:
I didn't mean to add both IPython and JupyterLab.

Here we're listing optional dependencies of PyGMT. IPython is, but JupyterLab is not. 

I think we should not make any changes here.

Reply:
Remove JupyterLab in 641c830.